src/Core/Users/login.class.php

```diff
@@ -0,0 +1,79 @@
+<?php
+require_once "UserInterface.php";
+
+Class login implements UserInterface{
+
+	public $_email;

```

Comment:
Private/protected variables used to be prefixed with an `_`, never public variables. Please change the naming conventions. For guidance, you can use `phpcs` with `psr2` as the standard. There should be a way of enabling `phpcs` in your komodo :P